src/modules/condition/ifcond.rs

```diff
@@ -91,7 +91,11 @@ impl SyntaxModule<ParserMetadata> for IfCondition {
 impl TranslateModule for IfCondition {
     fn translate(&self, meta: &mut TranslateMetadata) -> String {
         let mut result = vec![];
-        result.push(format!("if [ {} != 0 ]; then", self.expr.translate(meta)));
+        if self.expr.translate(meta).starts_with("\"") {
```

Comment:
I am not sure if this is something that we want to handle here.
Probably to fix the various cases in this PR is enough that the translate method automatically wrap a variable so this check is not needed.

@Ph0enixKM I am right?

Reply:
I added that check because it was generating `if [ ""$__AF_file_exist1_v0__28_8"" != 0 ]; thenif [ ""$__AF_file_exist1_v0__28_8"" != 0 ]; then` stuff like that with a double `"`

Reply:
i think its best to remove this check. if some part of code translates a thing unquoted, it should be fixed. if we add checks like this all over this place, it will be a giant mess

Reply:
Yeah that's my point but we can generate a bash like this so a check somewhere to not read them is needed

Reply:
so just do it like this without a check?

```rs
result.push(format!("if [ {} != 0 ]; then", self.expr.translate(meta)));
```

assuming that it should be translated quoted. if it's not translated quoted, then its an issue with the translator

Reply:
I think that https://github.com/amber-lang/amber/pull/283/ this PR implemented in some cases to wrap the variable but not everywhere so I am fixing in this PR in the wrong place.

Reply:
Don't do that check. Please find the root cause of what adds the quotes and fix it instead. If you need any help, let me know

Reply:
I wasn't able to find where it is happening, looking at the rest of changes I did in this PR there is somewhere where we escape but not like in all the cases.